apps/poker-roll/src/app/startSession.jsx

```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { TextField, Button, Stack } from "@mui/material";
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+
+const SessionForm = (props) => {
+    const [formState, setFormState] = useState({
+        buyin: "",
+        game: "",
+        casino: "",
+        location: "",
+      });
+
+      const handleChange = (event) => {
+        const { name, value } = event.target;
+    
+        setFormState({
+          ...formState,
+          [name]: value,
+        });
+      };
+    const [alignment, setAlignment] = React.useState('cashgame');
+
+  const handleToggleChange = (event, newAlignment) => {
+    setAlignment(newAlignment);
+  };
+  const handleFormSubmit = async (event) => {
+    event.preventDefault();
+
+    // form submit functionality
+};
+
+    return (
+        <div>
+            <form
+        className="bg-white m-4 border border-dark rounded text-dark newAccountForm"
+        onSubmit={handleFormSubmit}
+      >
+            <ToggleButtonGroup
+      color="primary"
+      value={alignment}
+      exclusive
+      onChange={handleToggleChange}
+      aria-label="Platform"
+    >
+      <ToggleButton value="cashgame">CashGame</ToggleButton>
+      <ToggleButton value="tournament">Tournament</ToggleButton>
+      </ToggleButtonGroup>
+      {/* change which stack is show depending on cashgame or tournament is selected */}
+
+      
+        <Stack
+          spacing={2}
+          direction="column"
+          justifyContent="center"
+          alignItems="center"
+        >
+          <h2>Enter New Tournament</h2>
+          <TextField
+            label="Tournamnet Buy In"
+            variant="outlined"
+            name="buyIn"
+            type="text"
+            value={formState.name}
+            onChange={handleChange}
+            size="small"
+            margin="normal"
+          />
```

Comment:
Minor casing issue, `name` is `buyIn` but state is `buyin`. Your `handleChange` grabs the `name` which won't match up with the state. Also `value` is using the `name` state here.

Reply:
believe this is fixed